app/services/katello/pulp3/repository.rb

```diff
@@ -262,8 +262,12 @@ def copy_version(from_repository)
         create_version(:base_version => from_repository.version_href)
       end
 
+      def version_zero?
+        repo.version_href.ends_with?('/versions/0/')
```

Comment:
I guess this works but... seems like should be a better way to identify the 0th version without parsing the path? (Pretty sure the pulp apis do not provide this.)

Reply:
We could fetch that version and check the "number" field, but i was trying to not add an additional call to pulp.